blocks/address/src/app.tsx

```diff
@@ -545,10 +584,32 @@ export const App: BlockComponent<RootEntity> = ({
                                 }}
                               />
                             ),
-                          }}
-                          sx={{
-                            display: "inline-flex",
-                            maxWidth: 420,
+                            sx: [
+                              (theme) => ({
+                                // The popover needs to know how tall this is to draw
+                                // a shadow around it
+                                height: TYPE_SELECTOR_HEIGHT,
+
+                                // Focus is handled by the options popover
+                                "&.Mui-focused": {
+                                  boxShadow: "none",
+                                },
+
+                                [`.${outlinedInputClasses.notchedOutline}`]: {
+                                  border: `1px solid ${theme.palette.gray[30]} !important`,
+                                },
+                              }),
+                              ...(open && suggestions.length
+                                ? [
+                                    popperPlacementInputNoRadius,
+                                    popperPlacementInputNoBorder,
+                                    {
+                                      borderRadiusBottomLeft: "0 !important",
+                                      borderRadiusBottomRight: "0 !important",
```

Comment:
Ideally we would reframe from using `!important` in favour of specificity, did you try using the relevant MUI classname to achieve the override using specificity? Or is there some other issue you ran into

Reply:
This will be moved to the design-system in the follow-up PR, so I suggest we leave it for now